packages/now-build-utils/src/fs/node-version.ts

```diff
@@ -42,15 +47,33 @@ export async function getSupportedNodeVersion(
         );
       }
     } else {
-      if (!silent) {
-        throw new Error(
-          'Found `engines` in `package.json` with an unsupported node range: ' +
-            engineRange +
-            '\nPlease use one of the following supported ranges: ' +
-            JSON.stringify(supportedOptions.map(o => o.range))
-        );
-      }
```

Comment:
Since `!silent` will be removed, does this mean that right now every node option can be used when it's active?

Reply:
`silent` is set to true when the version is forced via framework detection.

I removed it here because it doesn't make sense to conditionally throw.

If the version is unsupported we need to throw and stop the build immediately or else they will get a cryptic error message from AWS.

Reply:
Understood 👍